src/screen/Polkaassembly/PolkaassemblyDiscussionDetail.tsx

```diff
@@ -0,0 +1,173 @@
+import {RouteProp} from '@react-navigation/native';
+import {Text, useTheme} from '@ui-kitten/components';
+import moment from 'moment';
+import Icon from 'presentational/Icon';
+import {Label} from 'presentational/Label';
+import LoadingView from 'presentational/LoadingView';
+import Padder from 'presentational/Padder';
+import SafeView, {noTopEdges} from 'presentational/SafeView';
+import React from 'react';
+import {Linking, ScrollView, StyleSheet, TouchableOpacity, View} from 'react-native';
+import {usePolkaassemblyDiscussionDetail} from 'src/api/hooks/usePolkaassemblyDiscussionDetail';
+import {PolkaassemblyDiscussionStackParamList} from 'src/navigation/navigation';
+import globalStyles, {standardPadding} from 'src/styles';
+
+export function PolkaassemblyDiscussionDetail({
+  route,
+}: {
+  route: RouteProp<PolkaassemblyDiscussionStackParamList, 'PolkaassemblyDiscussionDetail'>;
+}) {
+  const theme = useTheme();
+  const id = route.params.id;
+  const {data} = usePolkaassemblyDiscussionDetail(id);
+
+  if (!data || !data.post) {
+    return <LoadingView />;
+  }
+
+  const {post, externalURL} = data;
+
+  return (
+    <SafeView edges={noTopEdges}>
+      <ScrollView style={styles.container} contentContainerStyle={styles.contentContainer}>
+        <Text category="h6">{post.title ?? ''}</Text>
+        <View style={styles.postDetailRow}>
+          <Text category="c2">{post.author?.username ?? ''} </Text>
+          <Text category="c1">posted in </Text>
+          <Label text={post.topic.name} />
+          <Text category="c1"> {moment(post.created_at).fromNow()}</Text>
+        </View>
+        <View style={styles.content}>
+          <Text style={styles.contentText}>{post.content?.trim() ?? ''}</Text>
+        </View>
+        <View style={styles.reactionRow}>
+          {post.comments.length ? (
+            <>
+              <View style={globalStyles.rowAlignCenter}>
+                <Icon name="message-circle-outline" style={globalStyles.icon15} fill="#ccc" animation="pulse" />
+                <Padder scale={0.3} />
+                <Text category="label" appearance="hint">
+                  {post.comments.length} comments
+                </Text>
+              </View>
+              <Padder scale={1} />
+            </>
+          ) : null}
+          <Text category="c1" appearance="hint">
+            👍 {post.likes.aggregate.count}{' '}
+          </Text>
+          <Text category="c1" appearance="hint">
+            {' '}
```

Comment:
Is this added by the editor when formatting or you need the extra space? if you need the extra space it would be  better to use template strings to add the space